images/Makefile

```diff
@@ -23,6 +23,7 @@ ARCH := $(shell uname -m)
 # tests are using locally-defined images (that are consistent and idempotent).
 REMOTE_IMAGE_PREFIX ?= gcr.io/gvisor-presubmit
 LOCAL_IMAGE_PREFIX ?= gvisor.dev/images
+ALL_ARCH ?= amd64 arm64
```

Comment:
Note that this is already possible today, which is why images are explicitly tagged with the arch. You can just do:

make push-all-images ARCH=aarch64

I'd rather keep this system as the "multi-arch" images are way too complex. I'd prefer if we set up the ARM64 CI system to build and push ARM64 images, that are explicitly tagged that way. Makes sense?